inventory.yaml

```diff
@@ -0,0 +1,65 @@
+vars:
+  registry: <registry>
+
+images:
+  - name: agent-ubuntu
+    vars:
+      context: .
+      template_context: scripts/dev/templates/agent
+
+    stages:
+      - name: agent-template-ubuntu
+        task_type: dockerfile_template
+        distro: ubuntu
+
+        inputs:
+          - agent_version
+          - tools_version
+          - tools_distro
+          - agent_distro
+
+        output:
+          - dockerfile: scripts/dev/templates/agent/Dockerfile.ubuntu-$(inputs.params.version_id)
+
+      - name: agent-ubuntu-build
+        task_type: docker_build
+        dockerfile: scripts/dev/templates/agent/Dockerfile.ubuntu-$(inputs.params.version_id)
+
+        labels:
+          quay.expires-after: 48h
+
+        output:
+          - registry: $(inputs.params.registry)/mongodb-agent-dev
```

Comment:
What about naming these registries in a more consistent way?

* `mongodb-agent-dev-ubuntu` and `mongodb-agent-dev-ubi` 

Instead of

* `mongodb-agent-dev` and `mongodb-agent-ubi-dev`

Reply:
yes that makes more sense! Will rename to keep things consistent.